arangodb-net-standard/GraphApi/Graph.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+
+namespace ArangoDBNetStandard.GraphApi
+{
+    public class Graph
```

Comment:
Until now we have created a specific version of this model for each method response, e.g. `PostGraphCreatedGraph` and `GetGraphsGraph`.

However they do seem to share most of the properties, except for `name` that is sometimes not there.

Should we use a single class named `Graph` to represent all those?

It's a bit difficult to have to find specific names otherwise. Like in that case, it could be `PostGraphEdgeDefinitionGraph`, which seems a bit too much.

@rossmills99 @AlistairB99124 What are your thoughts?

Reply:
We could use inheritance and simplify this file to:
`using System.Collections.Generic;

namespace ArangoDBNetStandard.GraphApi
{
    public class PostGraphEdgeDefinitionGraph : GraphBase
    {
        public string Name { get; set; }       

        public IEnumerable<EdgeDefinition> EdgeDefinitions { get; set; }
    }
}`

Reply:
I see you've updated the model name with `PostGraphEdgeDefinitionGraph`. I think that's good for now, let's discuss about this further in a separate issue. 👍